pkgs/development/compilers/julia/1.5.nix

```diff
@@ -33,13 +33,7 @@ stdenv.mkDerivation rec {
   };
 
   patches = [
-    ./use-system-utf8proc-julia-1.3.patch
-
-    # Julia recompiles a precompiled file if the mtime stored *in* the
-    # .ji file differs from the mtime of the .ji file.  This
-    # doesn't work in Nix because Nix changes the mtime of files in
-    # the Nix store to 1. So patch Julia to accept mtimes of 1.
-    ./allow_nix_mtime.patch
```

Comment:
Why exactly are we dropping it in 1.5 if the plan is just to keep 1.5 as a stop-gap?

Reply:
No huge reason, I just figured it was better to prune it as it had no real effect. Maybe an irrational fear that if I get hit by a bus it will stick around for a decade? ;)